website/server/models/user/schema.js

```diff
@@ -521,6 +521,7 @@ let schema = new Schema({
     return {};
   }},
   pushDevices: [PushDeviceSchema],
+  ABtest: {type: String},
```

Comment:
@crookedneighbor But I think it would make sense for any given user to be in **maximally one** AB test. Otherwise, the retention data wouldn't be particularly useful because there would be no way to tell which test/reason had an effect.  And as far as I can tell, the string wouldn't be checked in the code, as all of the code differences would take place during account/user creation.  Therefore, the field might as well be left as a (descriptive) string.